.vib/cert-manager/goss/cert-manager.yaml

```diff
@@ -0,0 +1,5 @@
+command:
+  # There are no syntax error in the configuration
```

Comment:
What do you mean by this comment?

Reply:
@migruiz4, a copy paste error. Fixed it now